horace_core/sqw/PixelData/@PixelData/private/check_pixel_fields_.m

```diff
@@ -0,0 +1,28 @@
+function check_pixel_fields_(obj, fields)
```

Comment:
Would this be a useful extension to general objects? I.e. make this a public function and replace references to "pixeldata" with `class(obj)` ?

Reply:
This function in particular is for `PixelData` data fields. These are tracked as the keys in a `containers.Map`, the map's values being the corresponding index/indices into the underlying data array. So I'm checking something more specific in this case.